GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -678,6 +677,19 @@ private void InternalInitialize(bool hard)
             }
         }
 
+        private void ReloadRepoObjectsTree()
+        {
+            if (IsRepoObjectsTreeVisible())
+            {
+                ThreadHelper.JoinableTaskFactory.RunAsync(() => repoObjectsTree.ReloadAsync()).FileAndForget();
+            }
+        }
+
+        private bool IsRepoObjectsTreeVisible()
+        {
+            return !MainSplitContainer.Panel1Collapsed;
```

Comment:
Is there a reason why we wouldn't inline this into `ReloadRepoObjectsTree`?

Reply:
Because `IsRepoObjectsTreeVisible` describes the intention behind the condition, while `!MainSplitContainer.Panel1Collapsed` does not. If you are a new contributor, it is easier to you understand what is going on reading `if (IsRepoObjectsTreeVisible)` than `!MainSplitContainer.Panel1Collapsed`.

Reply:
```c#
var isRepoObjectsTreeVisible = !MainSplitContainer.Panel1Collapsed;

if (isRepoObjectsTreeVisible) { ... }
```

Or make it a nested function to reduce the number of members on the type.

Reply:
I did the change, though to me it looks more natural when a local function operates on the enclosing function's local data. The same goes for local variable:
```csharp
var data = Module.GetData();
var isDataSpecial = data > 23 && data < 34;

if(isDataSpecial) { ... }
```

or

```csharp
var data = Module.GetData();
if(IsDataSpecial(data)) { ... }

return;

bool IsDataSpecial(int data)
{
  return data > 23 && data < 34;
}
```

But it is all a mater of taste to some degree.